packages/sommelier.rb

```diff
@@ -2,90 +2,197 @@
 
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.'
-  homepage 'https://chromium.googlesource.com/chromiumos/containers/sommelier'
-  version '1382ce084cc407'
+  homepage 'https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/'
+  version '20201025'
   compatibility 'all'
-  source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
-  source_sha256 'b58d799b16d20abf92369fe0749c73f7398996f0afa9933517051778a8bb16c3'
+  source_url 'https://chromium-review.googlesource.com/changes/chromiumos%2Fplatform2~2476815/revisions/5/patch?zip&path=%2FCOMMIT_MSG'
+  source_sha256 'd1850e1d4a1e1ec873b9e4add7a881e981f6c0bc17dfd2a1b85efd7df6dd84b4'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-1382ce084cc407-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '52c7940172194b786440ea9482835dfdab06df027e91e2e2ca1401acff397e94',
-     armv7l: '52c7940172194b786440ea9482835dfdab06df027e91e2e2ca1401acff397e94',
-       i686: '80c4866b627479b5469cfa04abecc297080b6b330ed9cf8484d2964ed11bfe94',
-     x86_64: '1b33ebc3d0201e1b700b4d55dc53951c77f5f1491af9c24e1c70e66d64107b40',
-  })
+# See merged at in source url to goto find actual tgz url used, which for some reason doesn't seem to have a stable sha256sum.
 
   depends_on 'mesa'
   depends_on 'xkbcomp'
-  depends_on 'xorg_server'
+  depends_on 'xorg_server' 
   depends_on 'psmisc'
+  depends_on 'vim' # Until there is a standalone package providing xxd
+  depends_on 'xdpyinfo'
+  depends_on 'xauth'
   depends_on 'xsetroot'
+  depends_on 'llvm' => :build
+
+  PEER_CMD_PREFIX = if ARCH == 'armv7l' || ARCH == 'aarch64' then '/lib/ld-linux-armhf.so.3' elsif ARCH == 'i686' then '/lib/ld-linux-i686.so.2' else '/lib64/ld-linux-x86-64.so.2' end
```

Comment:
This would be better done using `case` as follows:
```ruby
case ARCH
 when 'armv7l', 'aarch64'
    PEER_CMD_PREFIX='/lib/ld-linux-armhf.so.3'
 when 'i686'
    PEER_CMD_PREFIX='/lib/ld-linux-i686.so.2'
 when 'x86_64'
    PEER_CMD_PREFIX='/lib64/ld-linux-x86-64.so.2'
end
```

Reply:
Done.